src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -140,4 +135,13 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+    /**
+     * Returns an unmodifiable java List view with elements cast as immutable and sorted by name
+     *
+     */
+    public List<ReadOnlyPerson> sortedImmutableListView() {
+        internalList.sort(Comparator.comparing(o -> o.getName().toString()));
+        return Collections.unmodifiableList(internalList);
+    }
```

Comment:
Could consider cases where names are the same
Neat code! 👍

Reply:
Noted thanks!